conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. It is important to be vulnerable in your group so that they understand your strengths and weaknesses within coding. Being open can also allow your team members to help you with problems with your code that you did not catch or understand.
+- Retaining team members ideas. This is a form of respecting everyone's thoughts and opinions. This means to keep or reserve someones idea to let your teammates know that they are heard within the group and their opinions matter.
+- Humbleness. Being humble is a form of humility it is important for everyone in a team to understand that it is not about one person but that it is about the team.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Resentment. To resent means to feel anger or bitterness towards someone. This will disrupt the flow of the team and can lead to poor communication. This is not respecting the group.
+- Lack of Communication. Within a group, communication is key to keep the team on track. One way to practice good communication is by giving good criticism which is a form of respect.
```

Comment:
There is already a section concerning communication, so while this is a really good point, it might not fit here the way it's currently phrased. You could rephrase it to say something along the lines of how miscommunication and using inappropriate language escalates conflicts, which is why it's unacceptable.